delegation-toolkit/guides/delegation/use-delegation-scopes/constrain-scope.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Learn how to constrain a delegation scope using caveat enforcers.
+sidebar_label: Constrain a scope
+toc_max_heading_level: 3
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Constrain a delegation scope
+
+[Delegation scopes](index.md) define the delegation's initial authority and help prevent delegation misuse.
+You can further constrain these scopes and limit the delegation's authority by applying [caveat enforcers](../../../concepts/delegation/caveat-enforcers.md). 
+
+## Prerequisites
+
+- [Install and set up the Delegation Toolkit.](../../../get-started/install.md)
+- [Create a delegator account.](../execute-on-smart-accounts-behalf.md#3-create-a-delegator-account)
+- [Create a delegate account.](../execute-on-smart-accounts-behalf.md#4-create-a-delegate-account)
+- [Configure a delegation scope.](index.md)
+
+## Apply a caveat enforcer
+
+For example, Alice creates a delegation with an [ERC-20 transfer scope](spending-limit.md#erc-20-transfer-scope) that allows Bob to spend up to 10 USDC.
+If Alice wants to further constrain the scope so that Bob can use the delegation only once,
+she can apply the [`limitedCalls`](../../../reference/delegation/caveats.md#limitedcalls) caveat enforcer.
```

Comment:
I wonder if `limitedCalls` is the best example to use here - because the erc20 transfer scope is defining an allowance, `limitedCalls` with a limit of 1 is going to result in at least confusing behaviour (the scope suggests the delegate can use up to the allowance, across multiple calls, but limitedCalls says only once - so if they transfer less than the allowance the rest of it is locked).

I wonder if a timestamp enforcer (you can have this allowance, but you have to wait until Thursday), or another enforcer that would play a little nicer in this scenario would be less confusing.